backend-shared/config/db/managedenvironment_test.go

```diff
@@ -1,264 +1,134 @@
 
 import (
 	"context"
-	"strings"
-	"testing"
 
-	"github.com/stretchr/testify/assert"
-)
-
-func TestCreateManagedEnvironment(t *testing.T) {
-	SetupforTestingDB(t)
-	defer TestTeardown(t)
-
-	dbq, err := NewUnsafePostgresDBQueries(true, true)
-	if !assert.NoError(t, err) {
-		return
-	}
-	defer dbq.CloseDatabase()
-
-	ctx := context.Background()
-
-	clusterCredentials := ClusterCredentials{
-		Clustercredentials_cred_id:  "test-cluster-creds-test-3",
-		Host:                        "host",
-		Kube_config:                 "kube-config",
-		Kube_config_context:         "kube-config-context",
-		Serviceaccount_bearer_token: "serviceaccount_bearer_token",
-		Serviceaccount_ns:           "Serviceaccount_ns",
-	}
-
-	err = dbq.CreateClusterCredentials(ctx, &clusterCredentials)
-	assert.NoError(t, err)
-
-	managedEnvironment := ManagedEnvironment{
-		Managedenvironment_id: "test-managed-env-3",
-		Clustercredentials_id: clusterCredentials.Clustercredentials_cred_id,
-		Name:                  "my env101",
-	}
-	post := dbq.CreateManagedEnvironment(ctx, &managedEnvironment)
-	if !assert.NoError(t, post) {
-		return
-	}
-
-	getmanagedEnvironment := ManagedEnvironment{
-		Managedenvironment_id: "test-managed-env-3",
-		Clustercredentials_id: clusterCredentials.Clustercredentials_cred_id,
-		Name:                  "my env101",
-	}
-	get := dbq.GetManagedEnvironmentById(ctx, &getmanagedEnvironment)
-	if !assert.NoError(t, get) {
-		return
-	}
-	assert.Equal(t, managedEnvironment, getmanagedEnvironment)
-
-}
-
-func TestGetManagedEnvironmentById(t *testing.T) {
-
-	SetupforTestingDB(t)
-	defer TestTeardown(t)
-
-	dbq, err := NewUnsafePostgresDBQueries(true, true)
-	if !assert.NoError(t, err) {
-		return
-	}
-	defer dbq.CloseDatabase()
-
-	ctx := context.Background()
-
-	clusterCredentials := ClusterCredentials{
-		Clustercredentials_cred_id:  "test-cluster-creds-test-4",
-		Host:                        "host",
-		Kube_config:                 "kube-config",
-		Kube_config_context:         "kube-config-context",
-		Serviceaccount_bearer_token: "serviceaccount_bearer_token",
-		Serviceaccount_ns:           "Serviceaccount_ns",
-	}
-
-	err = dbq.CreateClusterCredentials(ctx, &clusterCredentials)
-	assert.NoError(t, err)
-
-	managedEnvironment := ManagedEnvironment{
-		Managedenvironment_id: "test-managed-env-4",
-		Clustercredentials_id: clusterCredentials.Clustercredentials_cred_id,
-		Name:                  "my env101",
-	}
-	post := dbq.CreateManagedEnvironment(ctx, &managedEnvironment)
-	if !assert.NoError(t, post) {
-		return
-	}
-
-	getmanagedEnvironment := ManagedEnvironment{
-		Managedenvironment_id: "test-managed-env-4",
-		Clustercredentials_id: clusterCredentials.Clustercredentials_cred_id,
-		Name:                  "my env101",
-	}
-	get := dbq.GetManagedEnvironmentById(ctx, &getmanagedEnvironment)
-	if !assert.NoError(t, get) {
-		return
-	}
-	assert.Equal(t, managedEnvironment, getmanagedEnvironment)
-
-	//check for inexistent primary key
```

Comment:
As mentioned above, I recommend preserving this 'inexistent primary key' test and moving it over to the Ginkgo version.